src/SerializeResult.cc

```diff
@@ -50,19 +50,21 @@ sos::Object WrapParseResultAST(snowcrash::ParseResult<snowcrash::Blueprint>& blu
     sos::Object object;
     snowcrash::Error error;
 
-    try {
-        object.set(SerializeKey::Version, sos::String(PARSE_RESULT_SERIALIZATION_VERSION));
-        object.set(SerializeKey::Ast, WrapBlueprint(blueprint, options.expandMSON));
-
-        if (options.exportSourceMap) {
-            object.set(SerializeKey::Sourcemap, WrapBlueprintSourcemap(blueprint.sourceMap));
+    if (blueprint.report.error.code == snowcrash::Error::OK) {
```

Comment:
TODO: Can revert this change because now WrapBlueprint will throw  and this is caught below.